File: curso_c_cpp/c/04_operadores/ejemplos/aritmeticos.c
/*
 * Operadores aritmeticos, con enfasis en / (division) y % (residuo).
 */
#include <stdio.h>

int main(void) {
    int a = 17, b = 5;
    printf("%d + %d = %d\n", a, b, a + b);
    printf("%d - %d = %d\n", a, b, a - b);
    printf("%d * %d = %d\n", a, b, a * b);
    printf("%d / %d = %d  (division entera)\n", a, b, a / b);
    printf("%d %% %d = %d  (residuo)\n", a, b, a % b);

    /* Precedencia: * antes que + */
    printf("2 + 3 * 4 = %d\n", 2 + 3 * 4);       // 14
    printf("(2 + 3) * 4 = %d\n", (2 + 3) * 4);   // 20
    return 0;
}

File: curso_java/04_operadores/ejemplos/Aritmeticos.java
/*
 * Operadores aritmeticos, con enfasis en / (division) y % (residuo).
 */
public class Aritmeticos {
    public static void main(String[] args) {
        int a = 17, b = 5;
        System.out.println(a + " + " + b + " = " + (a + b));
        System.out.println(a + " - " + b + " = " + (a - b));
        System.out.println(a + " * " + b + " = " + (a * b));
        System.out.println(a + " / " + b + " = " + (a / b) + " (division entera)");
        System.out.println(a + " % " + b + " = " + (a % b) + " (residuo)");

        System.out.println("2 + 3 * 4 = " + (2 + 3 * 4));       // 14
        System.out.println("(2 + 3) * 4 = " + ((2 + 3) * 4));   // 20
    }
}
